docs/marketplace/add_to_marketplace/manage_marketplace_items.md

```diff
@@ -95,46 +99,167 @@ The file contents will probably look like this:
       }
     }
   },
+  "supportedBy": "Acme Corporation Inc.",
   "supportedByImage": {
     "localPath": "./acmeCorporation.png"
   },
   "tenantId": "acme-corporation",
   "type": "plugin"
 }
 ```
+</details>
+
+You also want users to write services in your brand new programming language, `Acme.Js`.
+
+You want to provide a Template, which is a skeleton with a minimal setup: your users are required to write business code from scratch.
```

Comment:
I would better introduce both Templates and Examples and why you should use them in this example. 
Maybe, it would also be possible to redirect to the specific docs pages concerning Templates and Examples (and even enrich also them)